File: practiceForExam1/c/PROB10871.c
#include <stdio.h>

int main() {
    int N;
    int X;
    scanf("%d %d", &N, &X);

    int numbers[N];
    for (int i = 0; i < N; i ++) {
        scanf("%d", &numbers[i]);
        if ( numbers[i] < X)
            printf("%d ", numbers[i]);
    }

    return 0;
}

File: practiceForExam1/java/PROB10871.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.ArrayList;
import java.util.List;
import java.util.StringTokenizer;

public class PROB10871 {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static StringTokenizer st;
    static int N;
    static int X;
    static int[] numbers;
    static List<Integer> answers = new ArrayList<>();

    public static void main(String[] args) throws IOException {
        st = new StringTokenizer(br.readLine());
        N = Integer.parseInt(st.nextToken());
        X = Integer.parseInt(st.nextToken());

        numbers = new int[N];
        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < N; i++) {
            numbers[i] = Integer.parseInt(st.nextToken());
            if (numbers[i] < X) {
                System.out.print(numbers[i] + " ");
            }
        }
    }
}